packages/itwinui-react/src/core/ThemeProvider/ThemeProvider.tsx

```diff
@@ -336,6 +358,7 @@ const PortalContainer = React.memo(
     isInheritingTheme,
     theme,
     themeOptions,
+    future,
```

Comment:
Looks like we are not passing this prop currently, so all the dropdowns are showing up with v3 theme.